### ISSUE TITLE
Create a Fully Responsive Nav-Bar

### ISSUE THREAD

**Issue by saikatmu31** [websycode/Lets-Shop#5]
Fully Responsive Nav-Bar and Components Like Search Bar and Buttons and Dropdown Lists.

**Comment by saikatmu31** [comment]
I wish to Contribute to this issue.

**Comment by aakankshabhende** [comment]
Yes I'll assign it to you when coding period officially starts @saikatmu31
